cds_migrator_kit/rdm/migration/transform/models/thesis.py

```diff
@@ -0,0 +1,120 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of CERN Document Server.
+# Copyright (C) 2024 CERN.
```

Comment:
Should be 2025

Reply:
I didn't get the memo of a year change yet :)